config/symbols.pspeu.stst0.txt

```diff
@@ -1,4 +1,3 @@
-g_PauseAllowed = 0x091CE580;
```

Comment:
This is a global, have moved into `config/symbols.pspeu.txt`

Didn't spot any others like this